src/LibSDL2.jl

```diff
@@ -6267,411 +6270,411 @@ struct FPSmanager
 end
 
 function SDL_initFramerate(manager)
-    ccall((:SDL_initFramerate, libsdl2), Cvoid, (Ptr{FPSmanager},), manager)
+    ccall((:SDL_initFramerate, libsdl2_gfx), Cvoid, (Ptr{FPSmanager},), manager)
```

Comment:
Do all the `libsdl2_gfx` functions come from a specific header file? If so then we could set the library to be called for that header: https://github.com/JuliaInterop/Clang.jl/blob/master/gen/generator.toml#L12